rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -232,12 +219,6 @@ components:
           description: The size, in bytes, of the ingested product.
           minimum: 1
           example: 2147483648
-        callbackUrl:
-          type: string
-          description: The URL to call for each completed transformation and for the final completed transformation message.
-          minLength: 1
-          maxLength: 2048
-          example: https://www.example.com/mis
     TransformResponse:
```

Comment:
❗️Looks like we forgot to update the response. Since the post returns a `202` status code, we should expect the post to eventually return a URL that can be used to get a status on the transformation, cancel it, etc. For now, should update this and return an empty JSON object instead of returning information we know needs to be removed?